### PR TITLE
Replace placeholder of ZDB resource with "ZDB-Ressource"

### PR DESCRIPTION
**Pull request by ChristophEwertowski**
Resolves https://github.com/hbz/lobid-resources/issues/511

### CODE REVIEW

**Comment by ChristophEwertowski** [comment]
Deployed to test. For example, see [HT008359164 (test, json)](http://test.lobid.org/resources/HT008359164.json).

**Comment by acka47** [comment]
+1

**Comment by ChristophEwertowski** [comment]
@dr0i: I can't remember what you said I should do here.

**Comment by dr0i** [comment]
I think it might work to just use `@idzdb` instead of introducing the `@zdbUrl`.  Basically as it was before.

**Comment by ChristophEwertowski** [comment]
I removed `@zdbUrl` and took `@idzdb` again. One line more but easier to understand. Also merged the master branch in the development branch so that the conflicts are resolved.

File: src/main/resources/morph-hbz01-to-lobid.xml [hunk 1]
```diff
@@ -1022,6 +1022,15 @@
 		</combine>
 		<call-macro name="addTypeCollection" field="@idzdb"/>
 		<call-macro name="setBackRdfSubject" field="025z[-1].a"/>
+		<combine name="~rdf:subject" value="${a}">
+			<data source="@idzdb" name="a">
+				<regexp match="(.*)" format="$[ns-zdb-services]${1}"/>
+			</data>
```
Comment (dr0i) [inline src/main/resources/morph-hbz01-to-lobid.xml]: No need for the regex because this is already done in line 1012.

Comment (ChristophEwertowski) [reply src/main/resources/morph-hbz01-to-lobid.xml]: In line 1012 onwards the ZDB-URL is created for owl/sameAs. But it's not created to use elsewhere, so `@idzdb` still contains only the ZDB-Id and not the ZDB-URL. Therefore the regular expression is needed at line 1025ff to reference the newly built owl/sameAs-ZDB-URL.

**Comment by ChristophEwertowski** [comment]
@dr0i : Merged the master branch into the development branch and answered to your remark.

**Comment by dr0i** [comment]
Deployed to production, see e.g. lobid.org/resources/HT008359164.json.
Closed.

**Pull request merged**